fix(apple): Clarification for manual transactions (#3532)
getsentry/sentry-docs: src/platforms/common/performance/index.mdx

With [performance monitoring](/product/performance/), Sentry tracks your software performance, measuring metrics like throughput and latency, and displaying the impact of errors across multiple systems. Sentry captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services. Learn more about our model in [Distributed Tracing](/product/performance/distributed-tracing/).

<PlatformSection supported={["apple"]}>

> The `sentry-cocoa` SDK doesn't yet support automatic instrumentation for monitoring the performance of your application. Instead, you have to manually capture transactions.

Reviewer: Let's make this more generic, since other SDKs may encounter this same issue. 
```suggestion
Automatic instrumentation for monitoring the performance of your application isn't yet supported. Instead, you can <PlatformLink to="/performance/custom-instrumentation/">use custom instrumentation to capture transactions</PlatformLink>.
```
